include/nuttx/board.h

```diff
@@ -837,6 +837,27 @@ void board_init_rngseed(void);
 int board_reset_cause(FAR struct boardioc_reset_cause_s *cause);
 #endif
 
+/****************************************************************************
+ * Name:  board_ws2812_initialize
+ *
+ * Description:
+ *   This function may called from application-specific logic during its
+ *   to perform board-specific initialization of the ws2812 device
+ *
+ *
+ ****************************************************************************/
+
+#  ifdef CONFIG_WS2812
+#    ifndef CONFIG_WS2812_NON_SPI_DRIVER
+int board_ws2812_initialize(int devno, int spino, uint16_t nleds);
+#    else
+int board_ws2812_initialize(
+    int devno, 
+    uint16_t nleds, 
+    struct rmt_dev_s *rmt_dev);
```

Comment:
since rmt is specific for esp32, it's wrong to reference in the common header file.

Reply:
They are defined on `esp32-devkitc.h` header as well, so these may be simply removed.